locales/locales/en.json

```diff
@@ -3063,5 +3063,61 @@
 
     "gfycatUseMobileGfycatDesc": {
         "message": "Use mobile (lower resolution) gifs from gfycat."
+    },
+
+    "selectedEntryAutoSelectOnScrollDesc": {
+        "message": "Automatically select the topmost item while scrolling"
+    },
+
+    "selectedEntrySelectThingOnLoadDesc": {
+        "message": "Automatically select a post/comment when the page loads"
+    },
+
+    "selectedEntrySelectLastThingOnLoadDesc": {
+        "message": "Automatically select the last thing you had selected"
+    },
+
+    "selectedEntryScrollToSelectedThingOnLoadDesc": {
+        "message": "Automatically scroll to the post/comment that is selected when the page loads"
+    },
+
+    "selectedEntrySelectOnClickDesc": {
+        "message": "Allows you to click on an item to select it"
+    },
+
+    "selectedEntryAddFocusBGColorDesc": {
+        "message": "Use a background color"
+    },
+
+    "selectedEntryFocusBGColorDesc": {
+        "message": "The background color"
+    },
+
+    "selectedEntryFocusBGColorNightDesc": {
+        "message": "The background color while using Night Mode"
+    },
+
+    "selectedEntryFocusFGColorNightDesc": {
+        "message": "The text color while using Night Mode"
+    },
+
+    "selectedEntryAddFocusBorderDesc": {
+        "message": "Use a border"
+    },
+
+    "selectedEntryFocusBorderDesc": {
+        "message": "Border appearance. E.g. <code>1px dashed gray</code> (CSS)"
```

Comment:
It does work but we want to avoid it because eventually we will switch module/option descriptions to markdown (I should write that down somewhere).

In the case of something like a `<table>` you can just leave it untranslated for now and we'll replace all of those at the end; in this case I changed it to ````1px dashed gray```` because it looks fine both raw: \`1px dashed gray\` and in markdown: `1px dashed gray`.